wd_helper.au3

```diff
@@ -449,6 +449,40 @@ Func _WD_WaitElement($sSession, $sStrategy, $sSelector, $iDelay = Default, $iTim
 	Return SetError(__WD_Error($sFuncName, $iErr, $sParameters), 0, $sElement)
 EndFunc   ;==>_WD_WaitElement
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_DebugSwitch
+; Description ...: Switch to new debug level or switch back to saved debug level
+; Syntax ........: _WD_DebugSwitch([$vMode = Default])
+; Parameters ....: $vMode               - [optional] Set new $_WD_DEBUG level. When not specified (Default) restore saved debug level.
+; Return values .: Success - 1
+;                  Failure - 0
+; Author ........: mLipok
+; Modified ......:
+; Remarks .......: Function saves debug level at first call.
+;                  The first stored value will be never deleted, will be stored on the stack forever.
+; Related .......:
+; Link ..........:
+; Example .......: _WD_DebugSwitch($_WD_DEBUG_Full)
+; ===============================================================================================================================
+Func _WD_DebugSwitch($vMode = Default, $iErr = @error, $iExt = @extended)
+	Local Const $sFuncName = "_WD_DebugSwitch"
+	Local Static $a_WD_DEBUG_SavedStack[1] = [$_WD_DEBUG] ; at first run save currently used debug level to the stack
+	Local $iResult = 0
+
+	If $vMode = Default Then ; restoring saved debug level
+		$_WD_DEBUG = $a_WD_DEBUG_SavedStack[UBound($a_WD_DEBUG_SavedStack) - 1] ; restore last element on the stack
+		; check and do not delete stored debug level if this is the first one on the stack
+		If UBound($a_WD_DEBUG_SavedStack) > 1 Then ReDim $a_WD_DEBUG_SavedStack[UBound($a_WD_DEBUG_SavedStack) - 1]
+		$iResult = 1
+	ElseIf IsInt($vMode) And $vMode < $_WD_DEBUG_None And $vMode > $_WD_DEBUG_Full Then ; setting new debug level
```

Comment:
Are you sure about these comparisons? They look wrong to me.

Reply:
please take a look: [fix $vMode checking](https://github.com/Danp2/au3WebDriver/pull/406/commits/ba448b225cc125d1f210a35bff6c4b3caab08999)

Reply:
I still think this is wrong. It should be `>= $_WD_DEBUG_None` and `<= $_WD_DEBUG_Full`

Reply:
Oh my old eyes. Of course you have right.
Take a look again: [fixing $vMode checking](https://github.com/Danp2/au3WebDriver/pull/406/commits/cf5a231c7ea533248cdee5cbd9b3a7f8eef49379)